data/src/main/java/co/nimblehq/compose/crypto/data/model/response/CoinItemResponse.kt

```diff
@@ -72,7 +71,7 @@ data class CoinItemResponse(
     )
 }
 
-@Suppress("ComplexMethod")
```

Comment:
I think we could get rid of `"ComplexMethod"` by creating extension 😄 

Refer: https://detekt.dev/docs/rules/complexity/#complexmethod

```
fun BigDecimal?.orZero() = BigDecimal.ZERO
fun Double?.orZero() = 0.0
fun Int?.orZero() = 0
```